test/cli.js

```diff
@@ -59,15 +59,15 @@ describe('cli', function() {
   });
 
   it('should output stylelint warnings', function(done) {
```

Comment:
hm, `test/cli.js` currently is really more of an integration test suite, no? 
so imho its ok to test all i/o stuff there. 
it would probably be cleaner to eg remove the `-i` options out of unrelated cases, though